lua/telescope/pickers.lua

```diff
@@ -716,8 +716,14 @@ function Picker:delete_selection(delete_cb)
     return x > y
   end)
   for _, index in ipairs(selection_index) do
-    local selection = table.remove(self.finder.results, index)
-    delete_cb(selection)
+    -- WIP: Trying to implement logic which will not remove the item from selection if buffer deletion failed
+    --
+    local selection = self.finder.results[index]
+    local no_error, function_return = pcall(delete_cb, selection)
+    if not no_error then
+        local selection = table.remove(self.finder.results, index)
+      return
```

Comment:
We should not return here! We are in a loop of maybe 10 multiselected items. And i expect that all are removed on delete, even if the first one can't be closed (or something like this)